### PR TITLE
Feature/python pipelines fixed venv

### PR DESCRIPTION
**Pull request by lpinon**

### CODE REVIEW

**Comment by lpinon** [comment]
Needs to merge [cloud_run_deploy](https://github.com/ultymatom/hangar/commit/c766f1577b3b520a30c34cdf5a8a6338816eb2a9) first. This is only a fix.

File: scripts/pipelines/common/templates/deploy-cloud-run/deploy-cloud-run.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        l) awsRegion=${OPTARG};;
+        n) serviceName=${OPTARG};;
+        g) gCloudRegion=${OPTARG};;
+        t) port=${OPTARG};;
+        *) echo "Error: Unexpected flag." >&2
+            exit 1;;
+    esac
+done
+
+. "$(dirname "$0")/package-extra.sh"
```
[the remark this answers is not in the document]
Comment (ultymatom) [reply scripts/pipelines/common/templates/deploy-cloud-run/deploy-cloud-run.sh:24]: @sonatype-lift ignore

File: scripts/pipelines/common/templates/deploy-cloud-run/deploy-cloud-run.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+# we get what is located after the last '/' in the branch name, so it removes /ref/head or /ref/head/<folder> if your branche is named correctly"
+branch_short=$(echo "$branch" | awk -F '/' '{ print $NF }')
+
+# We change the name of the tag depending if it is a release or another branch
+echo "$branch" | grep release && tag_completed="${tag}"
```
[the remark this answers is not in the document]
Comment (ultymatom) [reply scripts/pipelines/common/templates/deploy-cloud-run/deploy-cloud-run.sh:29]: @sonatype-lift ignore

**Comment by ultymatom** [comment]
Changes applied in the deploy in cloud run

**Comment by ultymatom** [comment]
Changes applied in the deploy in cloud run

**Pull request closed without merging**